app/modules/users/permissions/rules.py

```diff
@@ -85,11 +85,11 @@ def check(self):
         # This Rule is for checking permissions on modules, so there must be one,
         assert self._module is not None
 
-        # Anonymous users can only create a submission or themselves
+        # Anonymous users can create: a submission, encounter, [sighting,] or themselves
         if not current_user or current_user.is_anonymous:
             has_permission = False
             if self._action == AccessOperation.WRITE:
-                has_permission = self._is_module(Submission) or self._is_module(User)
+                has_permission = self._is_module(Submission) or self._is_module(User) or self._is_module(Encounter)
```

Comment:
My initial fault. I thought I'd cleared these up but this should be self._is_module((Submission, User, Encounter))
See the _can_user function for examples.

Reply:
Plus you need to import Encounter which is why the lint and tests are failing